lib/gui-utils/src/GUIUtils.cpp

```diff
@@ -93,103 +118,292 @@ void GUIUtils::displayEndLine(String msg) {
   u8g2.print(msg.c_str());
 }
 
-void GUIUtils::displaySensorAvarage(int avarage) {
+void GUIUtils::displaySensorAvarage(int avarage)
+{
+#ifndef Emoticons
+#ifdef TTGO_TQ
+  if (avarage < 13)
+  {
+    u8g2.setFont(u8g2_font_unifont_t_emoticons);
+    u8g2.drawGlyph(76, 12, 0x0024);
+    u8g2.setFont(u8g2_font_5x7_tf);
+    u8g2.setCursor(77, 16);
+    u8g2.print("GOOD");
+  }
+  else if (avarage < 36)
+  {
+    u8g2.setFont(u8g2_font_unifont_t_emoticons);
+    u8g2.drawGlyph(76, 12, 0x0062);
+    u8g2.setFont(u8g2_font_4x6_tf);
+    u8g2.setCursor(77, 17);
+    u8g2.print("MODERATE");
+  }
+  else if (avarage < 56)
+  {
+    u8g2.setFont(u8g2_font_unifont_t_emoticons);
+    u8g2.drawGlyph(76, 12, 0x0032);
+    u8g2.setFont(u8g2_font_4x6_tf);
+    u8g2.setCursor(77, 17);
+    u8g2.print("UNHEALT S");
+  }
+  else if (avarage < 151)
+  {
+    u8g2.setFont(u8g2_font_unifont_t_emoticons);
+    u8g2.drawGlyph(76, 12, 0x0051);
+    u8g2.setFont(u8g2_font_4x6_tf);
+    u8g2.setCursor(77, 17);
+    u8g2.print("UNHEALTY");
+  }
+  else if (avarage < 251)
+  {
+    u8g2.setFont(u8g2_font_unifont_t_emoticons);
+    u8g2.drawGlyph(76, 12, 0x0053);
+    u8g2.setFont(u8g2_font_4x6_tf);
+    u8g2.setCursor(77, 17);
+    u8g2.print("VERY UNH");
+  }
+  else
+  {
+    u8g2.setFont(u8g2_font_unifont_t_emoticons);
+    u8g2.drawGlyph(76, 12, 0x0057);
+    u8g2.setFont(u8g2_font_4x6_tf);
+    u8g2.setCursor(77, 17);
+    u8g2.print("HAZARDOUS");
+  }
+#else
+  delay(1);
+#endif
+#else
+  if (avarage < 13)
+  {
+#ifdef TTGO_TQ
+    u8g2.drawXBM(1, 0, 32, 32, Smileface1);
+    u8g2.setFont(u8g2_font_6x12_tf);
+    u8g2.setCursor(40, 15);
+    u8g2.print("BUENO/");
```

Comment:
We have two lenguajes here? spanish and english? I think that we have many sections that need possible support for multiple lenguajes, and for size reasons maybe we only should support to english lenguaje

Reply:
Resolved:
https://github.com/danielbernalb/esp32-hpma115s0/commit/e15cc4bab6ad5edeec8961e4e06e03d533c9c7d5